acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/dsl/Eea.java

```diff
@@ -46,6 +47,6 @@ public String sendTransaction(final PrivateTransaction transaction) throws IOExc
 
   public Optional<PrivateTransactionReceipt> getTransactionReceipt(final String hash)
```

Comment:
Think this should be in another class a Besu class as it is a Besu specific api, whilst the other one is a Eea method

Reply:
There is now an Eea and a Besu